### ISSUE TITLE
extract-zip 1.1.0 causes ENOENT error when packaging for Windows/Linux

### ISSUE THREAD

**Issue by Tsur** [maxogden/electron-packager#133]
Hi guys, 

not sure if you'll be able to reproduce this error, but this is what I'm getting after setting up a fresh installation of electron-packager:

```bash
npm i electron-packager
electron-packager@5.1.0 node_modules/electron-packager
├── mv@2.1.1
├── rcedit@0.3.0
├── minimist@1.2.0
├── ncp@2.0.0
├── mkdirp@0.5.1 (minimist@0.0.8)
├── run-series@1.1.2 (dezalgo@1.0.3)
├── extract-zip@1.1.0 (debug@0.7.4, async@0.9.0, mkdirp@0.5.0, yauzl@2.3.1, concat-stream@1.5.0)
├── asar@0.6.1 (chromium-pickle-js@0.1.0, commander@2.3.0, glob@5.0.14, cuint@0.1.5, minimatch@2.0.4)
├── rimraf@2.4.3 (glob@5.0.14)
├── electron-download@1.1.0 (path-exists@1.0.0, home-path@0.1.2, debug@2.2.0, nugget@1.5.4)
└── plist@1.1.0 (util-deprecate@1.0.0, base64-js@0.0.6, xmldom@0.1.19, xmlbuilder@2.2.1)
```

Then, when I try to make it to work for linux (x64), this is what I get:

```
electron-packager . app --platform='linux' --arch='x64' --version=0.32.1 --out=.

Packaging app for platform linux x64 using electron v0.32.1
[ { [Error: ENOENT, mkdir '/tmp/electron-packager/linux-x64/IKABO-0.1.0-linux-x64/resources/app']
    errno: -2,
    code: 'ENOENT',
    path: '/tmp/electron-packager/linux-x64/IKABO-0.1.0-linux-x64/resources/app' } ] undefined

```

I made a bit of further research on this and figured out that the issue was coming from `extract-zip` dependence. When it upgrades to 1.1.0, it fails, but when rolling back to 1.0.3, it works again. Here the  line https://github.com/maxogden/electron-packager/blob/master/package.json#L22, so just using previous version, it gets fixed (note I removed ^ to bypass the upgrading):

```js
...
"extract-zip": "1.0.3",
...
```

Sure this is just a quick workaround, but meantime it's properly solved, it works. Note also this is not working from Linux, but I ignored about Win or OSX. Here my machine info(Ubuntu 14.04 LTS):

```
Linux 3.13.0-48-generic #80-Ubuntu SMP Thu Mar 12 11:16:15 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
``` 

This is perhaps related to https://github.com/maxogden/extract-zip/issues/10

Thanks

**Comment by maxogden** [comment]
should be fixed now (with extract-zip 1.1.1). sorry about that, tested with the MacOS zips but not the Linux ones, will do so in the future

**Comment by Tsur** [comment]
tested with 1.1.1 and works, nice job, thanks!
